test/levels.test.js

```diff
@@ -0,0 +1,92 @@
+import _ from 'lodash';
+import levels from '../src/levels';
+
+describe('levels', () => {
+  it('should be defined', () => {
+    let expectedLevels = [
+      'alert',
+      'critical',
+      'debug',
+      'emergency',
+      'error',
+      'fatal',
+      'info',
+      'informational',
+      'notice',
+      'silly',
+      'trace',
+      'verbose',
+      'warn',
+      'warning'];
+
+    _.forEach(expectedLevels, (lvl) => {
+      expect(levels[lvl]).toBeDefined();
+    });
+  });
+});
+
+describe('levelToLevelName', () => {
+  it('should be defined', () => {
+    expect(levels.levelToLevelName).not.toBeUndefined();
+  });
+
+  it('should return as expected for defined levels', () => {
+    expect(levels.levelToLevelName(levels.alert)).toBe('alert');
+    expect(levels.levelToLevelName(levels.critical)).toBe('critical');
+    expect(levels.levelToLevelName(levels.debug)).toBe('debug');
+    expect(levels.levelToLevelName(levels.emergency)).toBe('fatal');
+    expect(levels.levelToLevelName(levels.error)).toBe('error');
+    expect(levels.levelToLevelName(levels.fatal)).toBe('fatal');
+    expect(levels.levelToLevelName(levels.info)).toBe('info');
+    expect(levels.levelToLevelName(levels.informational)).toBe('info');
+    expect(levels.levelToLevelName(levels.notice)).toBe('notice');
+    expect(levels.levelToLevelName(levels.silly)).toBe('silly');
+    expect(levels.levelToLevelName(levels.trace)).toBe('trace');
+    expect(levels.levelToLevelName(levels.verbose)).toBe('debug');
+    expect(levels.levelToLevelName(levels.warn)).toBe('warn');
+    expect(levels.levelToLevelName(levels.warning)).toBe('warn');
+  });
+
+  it('should return as expected for custom levels', () => {
```

Comment:
"as expected" means that whatever you wrote on line 51 is correct, even if it would say "expect(true).toBe(true)".

please be more specific e.g. a level-based string